server/src/controllers/seasonController.ts

```diff
@@ -40,19 +34,30 @@ export const seasonController = Router();
  *       500:
  *         description: Internal Server Error. An error occurred while processing the request.
  */
-seasonController.post('/', async (req: Request, res: Response) => {
+seasonController.get('/', async (req: Request, res: Response) => {
   try {
-    const season = await prisma.seasons.create({
-      data: {
-        name: req.body.name,
-        startdate: req.body.startdate,
-        enddate: req.body.enddate,
-        imageurl: req.body.imageurl,
-      },
+    const filters: any = {
+      deletedat: null,
+    };
+    if (req.query.current) {
+      const today = new Date();
+      const formattedDate =
+        today.getFullYear() * (today.getMonth() < 9 ? 10000 : 100000) +
+        (today.getMonth() + 1) * 100 +
+        today.getDate();
```

Comment:
I think this is used in a couple places. Might be a good idea to abstract it?

Reply:
Updated